src/volue/mesh/examples/create_physical_timeseries.py

```diff
@@ -0,0 +1,27 @@
+import helpers
+
+from volue.mesh import Connection, Timeseries
+
+
+def main(address, port, root_pem_certificate):
+    """Showing how to create a physical time series resource."""
+
+    connection = Connection(address, port, root_pem_certificate)

```

Comment:
Could you create the connection the way it was introduced in https://github.com/Volue-Public/energy-mesh-python/pull/481?